presto-yarn-package/src/main/resources/appConfig-test-no-catalog.json

```diff
@@ -13,6 +13,8 @@
     "site.global.presto_query_max_memory": "5GB",
     "site.global.presto_query_max_memory_per_node": "600MB",
```

Comment:
I will merge/cleanup some of these test appConfigs once all the patches go in-+.